WMS/eez_percent_distribution_tutorial.ipynb

```diff
@@ -1768,16 +1759,60 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 55,
+   "execution_count": 34,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ReadTimeout",
+     "evalue": "HTTPSConnectionPool(host='allencoralatlas.org', port=443): Read timed out. (read timeout=30)",
+     "output_type": "error",
+     "traceback": [
+      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
```

Comment:
I'm not sure if this actually matters, but is the merge of error output intentional?

Reply:
Good point.  I should clear the output before committing ... also looking into adding https://pypi.org/project/nbstripout/ and a pre-commit hook.